service-front/assets/js/opg/session-timeout-dialog.js

```diff
@@ -116,24 +116,40 @@ var SessionTimeoutDialog = function (options) {
         this.releaseNavigation();
     };
 
-    this.trapNavigation = function () {
-        continueButton.keydown(function (e) {
-            if (e.key === 'Tab' && e.shiftKey) {
-                e.preventDefault();
-                logoutButton.focus();
+    this.trapNavigation = function (focusables) {
+        var currentElementIndex = 0;
+        focusables[currentElementIndex].focus();
+
+        var lastFocusableIndex = focusables.length - 1;
+
+        this.element.keydown(function (e) {
+            // capture only tab events on this dialog
+            if (e.key !== 'Tab') {
```

Comment:
I think there may be some browser compatibility issues if you don't include the keycode option too `(e.key === 'Tab' || e.keyCode === 9);`

Reply:
Thanks @johnnolan, fixed in 8ab4fc7e4